src/style.css

```diff
@@ -0,0 +1,137 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 700;
+}
+
+html {
+  max-width: 100%;
+  height: 100vh;
+  padding: 0;
+}
+
+body {
+  max-width: 100%;
+  height: 100vh;
+  padding: 0;
+  padding-top: 0;
+  margin-top: 0;
+}
+
+.search {
+  margin-top: 20px;
+  height: 70px;
+  box-sizing: border-box;
+}
+
+.search--small {
+  margin-top: 20px;
+  height: 44px;
+  box-sizing: border-box;
+}
+
+.input {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background: #fff url(images/Search.svg) no-repeat;
+  background-size: 19px 19px;
+  background-position: left 26px bottom 25px;
+  padding-left: 62px;
+}
+
+.input--small {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  width: 100%;
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background: url(images/Search.svg) no-repeat;
+  background-size: 11px 11px;
+  font-size: 14px;
+  background-position: left 12px bottom 15px;
+  padding-left: 33px;
+}
+
+.input:hover {
+  box-sizing: border-box;
```

Comment:
you can move this rule to * selector

Reply:
I don't understand tbh, what do u mean? I can move ".input:hover" inside of the "input"?